PolicyServicePkg/PolicyServicePkg.ci.yaml

```diff
@@ -36,6 +39,7 @@
             "MdePkg/MdePkg.dec",
             "MdeModulePkg/MdeModulePkg.dec",
             "PolicyServicePkg/PolicyServicePkg.dec",
+            "UnitTestFrameworkPkg/UnitTestFrameworkPkg.dec"

```

Comment:
I think you just need this for the tests? if so, you probably can move this to `"AcceptableDependencies-HOST_APPLICATION":` for better book-keeping.